docs/reference/esql/functions/mv_last.asciidoc

```diff
@@ -0,0 +1,27 @@
+[discrete]
+[[esql-mv_last]]
+=== `MV_LAST`
+[.text-center]
+image::esql/functions/signature/mv_last.svg[Embedded,opts=inline]
+
+Converts a multivalued field into a single valued field containing the last value. This is most
+useful when reading from a function that emits multivalued fields in a known order like <<esql-split>>:
+
+[source.merge.styled,esql]
+----
+include::{esql-specs}/string.csv-spec[tag=mv_last]
+----
+[%header.monospaced.styled,format=dsv,separator=|]
+|===
+include::{esql-specs}/string.csv-spec[tag=mv_last-result]
+|===
+
+The order that <<esql-multivalued-fields, multivalued fields>> are read from underlying storage is not
+guaranteed. It is *frequently* ascending, but don't rely on that. If you need the maximum field value
+use <<esql-mv_max>> instead of `MV_LAST`. `MV_MAX` has optimizations for sorted values so there isn't
+a performance benefit to `MV_LAST`. `MV_LAST` is really only useful with functions that create
```

Comment:
I'd tone the "really only" down ("mostly"?). I can imagine indexed MVs keyword fields and wanting to get the 1st/last value.

Reply:
Sure